.ci.yaml

```diff
@@ -19,27 +19,41 @@ platform_properties:
           {"dependency": "curl", "version": "version:7.64.0"}
         ]
   linux_android:
+    dimensions:
+      kvm: "1"
     properties:
       os: Ubuntu
       cores: "8"
       device_type: none
+      contexts: >-
+      [
+        "android_virtual_device"
+      ]
```

Comment:
Same here. We do not want to in the virtual device initialization/preparation for all linux_android targets.